assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -39,35 +39,140 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 	},
 
 	addStyleRules: function( styleControls, values, controls, placeholders, replacements ) {
-		var self = this,
-			dynamicParsedValues = self.getSettings( 'settingsModel' ).parseDynamicSettings( values, self.getSettings( 'dynamicParsing' ), styleControls );
+		var dynamicParsedValues = this.getSettings( 'settingsModel' ).parseDynamicSettings( values, this.getSettings( 'dynamicParsing' ), styleControls );
 
-		_.each( styleControls, function( control ) {
+		_.each( styleControls, ( control ) => {
 			if ( control.styleFields && control.styleFields.length ) {
-				self.addRepeaterControlsStyleRules( values[ control.name ], control.styleFields, controls, placeholders, replacements );
+				this.addRepeaterControlsStyleRules( values[ control.name ], control.styleFields, control.fields, placeholders, replacements );
 			}
 
 			if ( control.dynamic && control.dynamic.active && values.__dynamic__ && values.__dynamic__[ control.name ] ) {
-				self.addDynamicControlStyleRules( values.__dynamic__[ control.name ], control );
+				this.addDynamicControlStyleRules( values.__dynamic__[ control.name ], control );
 			}
 
 			if ( ! control.selectors ) {
 				return;
 			}
 
-			self.addControlStyleRules( control, dynamicParsedValues, controls, placeholders, replacements );
+			this.addControlStyleRules( control, dynamicParsedValues, controls, placeholders, replacements );
 		} );
 	},
 
 	addControlStyleRules: function( control, values, controls, placeholders, replacements ) {
-		ControlsCSSParser.addControlStyleRules(
-			this.stylesheet,
-			control,
-			controls,
-			( StyleControl ) => this.getStyleControlValue( StyleControl, values ),
-			placeholders,
-			replacements
-		);
+		var value = this.getStyleControlValue( control, values );
+
+		if ( undefined === value ) {
+			return;
+		}
+
+		_.each( control.selectors, ( cssProperty, selector ) => {
```

Comment:
It doesn't work for `multiple` based controls like font-size.

Reply:
It works fine for me